src/MapperMMC3.cpp

```diff
@@ -272,6 +286,14 @@ void MapperMMC3::scanline()
 	}	
 }
 
+void MapperMMC3::scanlineIRQ(){
+	scanline();
+	if(irqState()){
+		m_interruptCallback(int(CPU::InterruptType::IRQ));
```

Comment:
Why cast to int here instead of passing the enum itself?

Reply:
There will be a loop include between `CPU.h` and `Mapper.h`. Can I move the enum to a `Constant.h` so both of them can use it?

Reply:
Why would there be a loop? Mapper doesn't doesn't on CPU

Reply:
Done.